libs/MVS/SceneTexture.cpp

```diff
@@ -509,11 +519,31 @@ bool MeshTexture::ListCameraFaces(FaceDataViewArr& facesDatas, float fOutlierThr
 		faceMap.create(imageData.height, imageData.width);
 		depthMap.create(imageData.height, imageData.width);
 		RasterMesh rasterer(vertices, imageData.camera, depthMap, faceMap);
+		// creating mask for the image border
```

Comment:
as you update the cv::floodfill usage with mask, pls set the mask valid pixels to 255 and invalid pixels to 0; this will make easier the integration of PR #967